src/components/elements/popover/types.ts

```diff
@@ -1,9 +1,11 @@
 import { PopoverInlineProps } from "@lifesg/react-design-system/popover-v2";
 import * as Icons from "@lifesg/react-icons";
+import { ITextSchema, ITypographySchema } from "../text";
 import { IBaseElementSchema } from "../types";
+import { IInlineWrapperSchema } from "../wrapper";
 
 interface IPopoverHint extends Pick<PopoverInlineProps, "position" | "customOffset" | "zIndex"> {
-	content: string;
+	content: string | Record<string, ITextSchema | ITypographySchema | IInlineWrapperSchema>;
```

Comment:
we can follow `src/components/custom/filter/types.ts` to support TBlockElementSchema and the default TWrapperSchema for rendering of other FEE elements